R/repositories.R

```diff
@@ -68,11 +68,30 @@ BINARY_BASE_URL <- "https://bioconductor.org/packages/%s/container-binaries/%s"
     repos
 }
 
+.repositories_ci_mirror_envopt <-
+    function()
+{
+    .env_opt_lgl(
+        "BIOCMANAGER_USE_CI_MIRROR", "BiocManager.use_ci_mirror", TRUE
+    ) && as.logical(Sys.getenv("CI"))
+}
+
+.BIOCONDUCTOR_CI_MIRROR <- "https://ci-mirror.bioconductor.org"
```

Comment:
This is hard-coded so cannot be changed without updating the CRAN BiocManager package. Instead, consult https://bioconductor.org/config.yaml?

Reply:
Should we add the URL to the list of mirrors perhaps as `CI-Bioconductor` and under `0-Bioconductor`?

Reply:
The hard-coded constant has been removed. We now resolve the URL from https://bioconductor.org/config.yaml and look for they keywords **Bioconductor** and **CI** in the institution description.